### YamlMime:UniversalReference
items:
  - uid: '@azure/servicefabric.TestErrorChaosEvent'
    name: TestErrorChaosEvent
    fullName: TestErrorChaosEvent
    children:
      - '@azure/servicefabric.TestErrorChaosEvent.kind'
      - '@azure/servicefabric.TestErrorChaosEvent.reason'
      - '@azure/servicefabric.TestErrorChaosEvent.timeStampUtc'
    langs:
      - typeScript
    type: interface
    summary: |-
      <span data-ttu-id="4604b-101">Descreve um evento de caos que é gerado quando ocorre um evento inesperado no mecanismo de Chaos.</span><span class="sxs-lookup"><span data-stu-id="4604b-101">Describes a Chaos event that gets generated when an unexpected event occurs in the Chaos engine.</span></span>
      <span data-ttu-id="4604b-102">Por exemplo, devido ao instantâneo de cluster que está sendo inconsistentes, ao mesmo tempo em que a falha de uma entidade, o Chaos encontrado que a entidade já falha – que seria um evento inesperado.</span><span class="sxs-lookup"><span data-stu-id="4604b-102">For example, due to the cluster snapshot being inconsistent, while faulting an entity, Chaos found that the entity was already faulted -- which would be an unexpected event.</span></span>
    package: '@azure/servicefabric'
  - uid: '@azure/servicefabric.TestErrorChaosEvent.kind'
    name: kind
    fullName: kind
    children: []
    langs:
      - typeScript
    type: property
    summary: <span data-ttu-id="4604b-103">Discriminador polimórfico</span><span class="sxs-lookup"><span data-stu-id="4604b-103">Polymorphic Discriminator</span></span>
    syntax:
      content: 'kind: "TestError"'
      return:
        type:
          - '"TestError"'
    package: '@azure/servicefabric'
  - uid: '@azure/servicefabric.TestErrorChaosEvent.reason'
    name: reason
    fullName: reason
    children: []
    langs:
      - typeScript
    type: property
    summary: '<span data-ttu-id="4604b-104">Descreve por que TestErrorChaosEvent foi gerado.</span><span class="sxs-lookup"><span data-stu-id="4604b-104">Describes why TestErrorChaosEvent was generated.</span></span> <span data-ttu-id="4604b-105">Por exemplo, o Chaos tenta uma partição de falha, mas localiza a que a partição não é mais tolerante a falhas, então um TestErrorEvent seja gerado com o motivo informando que a partição não tolerante a falhas.</span><span class="sxs-lookup"><span data-stu-id="4604b-105">For example, Chaos tries to fault a partition but finds that the partition is no longer fault tolerant, then a TestErrorEvent gets generated with the reason stating that the partition is not fault tolerant.</span></span>'
    optional: true
    syntax:
      content: 'reason?: undefined | string'
      return:
        type:
          - undefined | string
    package: '@azure/servicefabric'
  - uid: '@azure/servicefabric.TestErrorChaosEvent.timeStampUtc'
    name: timeStampUtc
    fullName: timeStampUtc
    children: []
    langs:
      - typeScript
    type: property
    summary: <span data-ttu-id="4604b-106">O carimbo de hora de UTC quando esse evento Chaos foi gerado.</span><span class="sxs-lookup"><span data-stu-id="4604b-106">The UTC timestamp when this Chaos event was generated.</span></span>
    syntax:
      content: 'timeStampUtc: Date'
      return:
        type:
          - Date
    package: '@azure/servicefabric'